epochStart/dtos.go

```diff
@@ -0,0 +1,18 @@
+package epochStart
+
+import (
+	"math/big"
+
+	"github.com/ElrondNetwork/elrond-go/state"
+)
+
+// OwnerData is a struct containing relevant information about owner's nodes data
+type OwnerData struct {
+	NumStakedNodes  int64
+	NumActiveNodes  int64
+	NumAuctionNodes int64
```

Comment:
I think `NumAuctionNodes` can be removed because are duplicated information with the `len(AuctionList)`

Reply:
Right, good suggestion; removed it from here and from `stakingDataProvider`.
However, I've kept it in `auctionListSelector`, since there we do a lot of iterations and I don't want to re-calculate `len` every time.